articles/extensions/sso-dashboard.md

```diff
@@ -0,0 +1,75 @@
+---
+description: This page explains how to configure and utilize the SSO Dashboard Extension
+---
+
+# Auth0 Extension: Single Sign-On (SSO) Dashboard
+
+The **SSO Dashboard** extension allows you to create a dashboard with multiple enterprise applications that can be enabled for single sign-on for your users for login.
+
+[View this Extension on GitHub](https://github.com/auth0-extensions/auth0-sso-dashboard-extension)
+
```

Comment:
Before jumping into configuration we need to do 2 things:

 - Explain that 2 types of users can login. Dashboard Admins can login to configure the applications and end users can login to see the dashboard and login to the applications.
 - Explain that in order to achieve this we need to:
    - Create a client (https://auth0.com/docs/extensions/delegated-admin#create-a-client)
    - Enable connections for the client: https://auth0.com/docs/extensions/delegated-admin#enable-connections-on-the-client. This section should not be copy pasted from delegated admin. We just need to explain that every connection enabled for the client will allow users to login to the SSO dashboard
 - Explain that there will be 2 clients: https://auth0.com/docs/extensions/delegated-admin#install-the-extension
 - Explain that there are 2 urls
   - https://YOU.REGION.webtask.io/auth0-sso-dashboard/admins/login => where admins can login
   - https://YOU.REGION.webtask.io/auth0-sso-dashboard/admins/login => where end users can login